guides/source/form_helpers.md

```diff
@@ -238,7 +238,10 @@ TIP: Conventionally your inputs will mirror model attributes. However, they don'
 
 #### The `fields_for` Helper
 
-You can create a similar binding without actually creating `<form>` tags with the [`fields_for`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-fields_for) helper. This is useful for editing additional model objects with the same form. For example, if you had a `Person` model with an associated `ContactDetail` model, you could create a form for creating both like so:
+The [`fields_for`][] helper creates a similar binding but without rendering a
+`<form>` tag. This is useful for editing additional model objects with the same
```

Comment:
To me the following is just a bit clearer, but feel free to ignore.
```suggestion
`<form>` tag. This is useful for editing additional model objects within the same
```

Reply:
I see what you mean about "within" describing additional input fields in the same form.  Here, "with" is used in the sense of "by means of" (i.e. "editing additional model objects by means of the same form"), but "with" is a very overloaded word in English, so let's go *with* "within".  I also tweaked the wording a bit more to better fit "within".